src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.operation.Operation;
+import core.basesyntax.service.activity.BalanceHandler;
+import core.basesyntax.service.activity.OperationHandler;
+import core.basesyntax.service.activity.SupplyHandler;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class OperationStrategyImplTest {
+    @Test
+    public void getOperation_Ok() {
+        MapStrategy mapStrategy = new MapStrategy();
+        FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
make global field of them and initialize inside of a method with @BeforeClass

Reply:
:)